tests/MongoDB.Driver.Core.Tests/Core/Configuration/ClusterSettingsTests.cs

```diff
@@ -56,6 +57,14 @@ public void constructor_should_throw_when_serverSelectionTimeout_is_negative()
             action.ShouldThrow<ArgumentException>().And.ParamName.Should().Be("serverSelectionTimeout");
         }
 
+        [Fact]
+        public void constructor_should_throw_when_localThreshold_is_negative()
+        {
+            Action action = () => new ClusterSettings(localThreshold: TimeSpan.FromSeconds(-1));
+
+            action.ShouldThrow<ArgumentException>().And.ParamName.Should().Be("localThreshold");
```

Comment:
In general we want to use `FluentAssertions` for any new tests.

Reply:
Done.